src/components/failedTaskBanner/index.tsx

```diff
@@ -63,26 +53,27 @@ const countFailedProcesses = (processes: ProcessWithStatus[]): CountFailedProces
 
 export default function FailedTaskBanner() {
     const { runningProcesses, completedProcessIds, useFallback } = useContext(RunningProcessesContext);
```

Comment:
runningProcesses doesn't seem to be used anymore, so no updates with websockets

Reply:
It's still used as dependency in the useEffect hook